DemoApp-Java/app/src/main/AndroidManifest.xml

```diff
@@ -24,14 +20,29 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+
+        <activity
+            android:name=".ads.BannerAdActivity"
+            android:parentActivityName=".HomeActivity" />
         <activity
-            android:name="com.applovin.enterprise.apps.demoapp.ads.BannerAdActivity"
-            android:parentActivityName="com.applovin.enterprise.apps.demoapp.HomeActivity" />
+            android:name=".ads.LayoutEditorBannerAdActivity"
+            android:parentActivityName=".HomeActivity" />
         <activity
-            android:name="com.applovin.enterprise.apps.demoapp.ads.RewardedAdActivity"
-            android:parentActivityName="com.applovin.enterprise.apps.demoapp.HomeActivity" />
+            android:name=".ads.RewardedAdActivity"
+            android:parentActivityName=".HomeActivity" />
         <activity
-            android:name="com.applovin.enterprise.apps.demoapp.ads.InterstitialAdActivity"
-            android:parentActivityName="com.applovin.enterprise.apps.demoapp.HomeActivity" />
+            android:name=".ads.InterstitialAdActivity"
+            android:parentActivityName=".HomeActivity" />
+
+        <!-- AppLovin -->
+        <meta-data
+            android:name="applovin.sdk.key"
+            android:value="SDK_KEY_HERE" />
+
+        <!-- Google Ads for Google Advertising ID -->
+        <meta-data
```

Comment:
Is this needed?

Reply:
I thought I did, but on a fresh install it isn't, so I'll remove it

Reply:
I thought I did, but on a fresh install it wasn't necessary, so I'll remove it.